components/Link/Link.stories.tsx

```diff
@@ -16,21 +16,21 @@ export default {
 
 type Story = StoryObj<typeof Link>;
 
-export const button: Story = {
+export const Button: Story = {
   args: {
     appearance: "button",
     children: "Pedir juguete",
   },
 };
 
-export const link: Story = {
+export const Link: Story = {
```

Comment:
Podrias renombrar esta story de `Link` a algo como `Default`? 
Ahora mismo esta causando un conficto porque esta duplicada la variable 'Link'

```suggestion
export const Default: Story = {
```